x-pack/plugins/secops/public/components/load_more_table/index.tsx

```diff
@@ -28,18 +29,30 @@ export interface ItemsPerRow {
   numberOfRow: number;
 }
 
+export interface SortingBasicTable {
+  field: string;
+  direction: Direction;
+}
+
+export interface Criteria {
+  page: { index: number; size: number };
+  sort: { field: string; direction: 'asc' | 'desc' };
```

Comment:
There exists a `export type SortRequestDirection = 'asc' | 'desc';` type. Do we want to promote that type from server to general usage for both front and back?

Reply:
In addition to this and Frank's other comments on sorting `state`, do we want to provide the ability for a user to _undo_ sorting / return to the _default_ sort of a column? Oftentimes a column's `sort_direction` is toggled as `asc`->`desc`->`default` to allow the user to undo their changes and revert to the original table state. Just mentioning as this was something I tried to do when testing the table.

Similarly, in terms of persistence & user configuration, would we want to allow the user to reset a table/component to its default state? Right now `dnsSortField/Direction` and `isPtrIncluded` is stored in redux, and whatever the user sets them to will persist throughout the current session until the page is reloaded. This is probably a larger discussion around component `display properties` and _how configurable_ we want are components to be, but I wanted to bring this up as we start adding more user defined state to our components.

Reply:
It makes sense I will change all sorting to match eui away